src/ring_theory/power_series.lean

```diff
@@ -78,6 +78,7 @@ instance [add_monoid α]      : add_monoid      (mv_power_series σ α) := pi.ad
 instance [add_group α]       : add_group       (mv_power_series σ α) := pi.add_group
 instance [add_comm_monoid α] : add_comm_monoid (mv_power_series σ α) := pi.add_comm_monoid
 instance [add_comm_group α]  : add_comm_group  (mv_power_series σ α) := pi.add_comm_group
+instance [nontrivial α]      : nontrivial      (mv_power_series σ α) := fun.nontrivial
```

Comment:
There was another instance for `power_series`, right?

Reply:
Yes, just below (line 769 of this file). I have changed it too.